docs/admins/access-overrides.rst

```diff
@@ -1,51 +1,61 @@
 
-Access Override
-===============
+Access overrides
+================
+
+.. vale Google.Parens = NO
+.. vale Google.Passive = NO
+.. vale Google.Will = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+.. vale write-good.TooWordy = NO
+.. vale write-good.Weasel = NO
+
 An access override allows someone access to (mostly electronic) RoboJackets services before they pay dues for the current semester.
 Typically, access overrides are provided on a case-by-case basis upon request.
 
-Self-Service
+Self-service
 ------------
 
 .. seealso::
    There is a :doc:`separate member-facing page for self-service overrides</members/access-override>`.
 
-To prevent misuse, several criteria must be met before a user becomes eligible for a self-service override.
+To prevent misuse, a user must meet several criteria before they become eligible for a self-service override.
 A user's eligibility for a self-service override is visible on their details page in Nova, in the System Access panel.
 
 Self-service overrides can last between 7 and 60 days.
-The longest duration will occur if the next (earliest) future :ref:`Dues Package` :guilabel:`Access End Date` is 60 or more days in the future.
-The shortest duration will occur if the next future :guilabel:`Access End Date` is 7 or fewer days away.
+The longest duration will occur if the earliest future :ref:`dues package <Dues package>` :guilabel:`Access End Date` is 60 or more days in the future.
+The shortest duration will occur if the earliest future :guilabel:`Access End Date` is 7 or fewer days away.
 
 Self-service override criteria are divided into 2 categories: conditions and tasks.
-A condition is a system- or user-level property that cannot be rectified by the user alone (see list below). However, tasks can be completed by users.
+A condition is a system- or user-level property that a user can't rectify alone.
+A task must be completed by the user themselves.
 
-Required Conditions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Required conditions
+~~~~~~~~~~~~~~~~~~~
 
-- User access must not be active
+- User access must not be already active
 - User must have no prior dues payments
 - User must have no active or prior access override
-- A :ref:`Dues Package` with the :guilabel:`Access End Date` in the future must exist
+- A :ref:`dues package <Dues package>` with an :guilabel:`Access End Date` in the future must exist
 
-Required Tasks
+Required tasks
 ~~~~~~~~~~~~~~
 
 - Sign the membership agreement
 - Attend a team meeting
-   - Attendance must be recorded in MyRoboJackets for it to count.
-   - Only teams in MyRoboJackets that have the :guilabel:`Self-Service Override Eligible` property set to true in Nova will count towards this requirement.
-     This means that event attendance (e.g., General Interest meetings), training attendance, and Core or discipline Core attendance will not satisfy this requirement.
+   - An officer must record attendance in Apiary for it to count.
```

Comment:
I don't like the use of "officer" here, especially for admin-facing docs, we should be more specific about the exact requirements. something like "either directly or via a remote attendance link"